Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8-slim as BUILD
+WORKDIR /wheel
+COPY . .
+RUN pip3 wheel --wheel-dir=/wheel \
+    errbot errbot[irc] errbot[slack] errbot[XMPP] errbot[telegram]
+
+FROM python:3.8-slim
+COPY --from=BUILD /wheel /wheel
+RUN apt update && \
+    apt install -y git && \
+    cd /wheel && \
+    pip3 -vv install --no-index --find-links /wheel \
```

Comment:
It looks like you might also want `--no-cache-dir` to minimize space usage, https://pip.pypa.io/en/stable/reference/pip_install/#caching.

Reply:
Thanks for the suggestion!